MATLAB/sensor_reading.m

```diff
@@ -48,9 +48,15 @@
 
 sensor_readings.time= true_state.time;
 sensor_readings.position_ecef= position_ecef;
-%TODO get velocity in ECEF
+sensor_readings.velocity_ecef= utl_rotateframe(quat_ecef_eci, true_state.velocity_eci)-cross(rate_ecef,position_ecef);
 
 
+%in reality this is not how cd cps works
+target_position_eci= other_satellite_state.dynamics.position_eci;
+sensor_readings.target_position_ecef= utl_rotateframe(quat_ecef_eci,target_position_eci);
+target_velocity_eci= other_satellite_state.dynamics.velocity_eci;
+sensor_readings.target_velocity_ecef= utl_rotateframe(quat_ecef_eci, target_velocity_eci)-cross(rate_ecef,sensor_readings.target_position_ecef);
```

Comment:
Does CD GPS actually give us a velocity reading or were we going to calculate that ourselves?

Reply:
Yeah, we have to calculate that ourselves.

Reply:
So `sensor_readings.target_velocity_ecef` will only live in MATLAB?